src/components/Auction.jsx

```diff
@@ -15,14 +15,18 @@ const Auction = ({ id, name, expiration, highest_bid, image }) => {
                         src={image}
                         alt="#"
                     ></img>
-                )) || <p>no image</p>}
+                )) || (
+                    <img
+                        className="article__imgholder__img"
+                        src="../src/images/default.jpg"
+                        alt="default image"
+                    ></img>
```

Comment:
met een onError kan je de src aanpasses moest deze niet (of foutief) meegegeven worden
`<img
                        className="article__imgholder__img"
                        src={image}
                        alt="#"
                        onError={({currentTarget}) =>
                            (currentTarget.src = "../src/images/default.jpg")
                        }
                    />`